mbuild/utils/geometry.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from mbuild.coordinate_transform import angle
+
+def calc_dihedral(point1, point2, point3, point4):
```

Comment:
I had this thought as well. Actually originally I thought that perhaps I should have put `calc_dihedral` in `coordinate_transform.py` instead of `utils`. I had primarily been using this function for debugging and as it turns out the unit tests I wrote that rely on this function could actually probably be comparing the `xyz` positions of the `Compound` instead of the FCCF dihedral angle (making sure that the positions are unchanged when bonds are broken and reformed). However, the reason I kept the dihedral angle in the test is that if this code was to get broken, looking at the change in the dihedral angle rather than the full `xyz` coordinates makes it slightly easier to debug. 

So I'm not sure what the best solution is. Should we put `angle` in `geometry.py` in `utils`, or should we just move `calc_dihedral` to `coordinate_transform.py` (and rename to `dihedral_angle` as you've suggested)?